appengine/mailjet/app.js

```diff
@@ -60,14 +60,14 @@ app.post('/hello', function (req, res, next) {
   var request = Mailjet.post('send').request(options);
 
   request
-    .on('success', function (response, body) {
+    .then(function (response, body) {
```

Comment:
Nit: I take it you've tested this?

(If you have, LGTM.)

Reply:
It has been tested :) The new mailjet library uses Promise instead of Event Emitter.